src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
it handles the magic for us, until the official spec is validated and implemented everywhere :) (or not :p)
(but remove that ugly _ with `this.handleSubmit = this.handleSubmit.bind(this)`it's safe)